src/lib.rs

```diff
@@ -29,8 +187,11 @@ mod symbol_table;
 mod system_reader;
 mod writer;
 
+#[doc(inline)]
 pub use data_source::IonDataSource;
+#[doc(inline)]
 pub use raw_symbol_token::RawSymbolToken;
+#[doc(inline)]
```

Comment:
🗺️ `#[doc(inline)]` prevents these from showing up in a big "Re-exports" list in the docs. We'll probably add this to more types before 1.0.